### DISCUSSION TITLE
GST Details

### DISCUSSION THREAD

**Discussion by KrutikaMoorjani** [ONDC-Official/ONDC-FIS-Specifications#16] [Q&A]
**Sole Prop Flow**

1. Are GST details going to be fetched from AA? If yes, AA currently does not provide GSTR-2 , will this be taken care by GSP as shown in the Figma? 

2. As per figma, understanding is that buyer app will seek GST consent from borrower and also provide the complete GSTIN details of the borrower from GSPs to the lender, is this understanding correct?

3. Are we targeting borrowers who do not have any GSTIN ? Is ONDC protocol only for valid GSTIN borrowers?

**Comment by 92shreyansh** [comment]
@ Antriksh-Parmar

**Reply by Antriksh-Parmar** [reply]
@KrutikaMoorjani 
1- please refer to the BRD and the Product flow.
2- Yes
3- Yes, you can use PL flow for the borrower with no GSTIN. There's an option to share the UDYAM anyway so that the borrower can be eligible for future credit guarantee if eligible.

**Reply by KrutikaMoorjani** [reply]
For 3rd query , 
~ If the customer is an MSME without GSTIN, then will buyer app give him an option to enquire for BL product? As MSME customers would enquire for BL generally @Antriksh-Parmar 
~ We as a lender can even facilitate BL for non GSTIN MSME customers

**Reply by Antriksh-Parmar** [reply]
What are all the data points go into underwriting BL for MSMEs? You can reply here or drop me a note on antriksh.parmar@ondc.org

**Reply by KrutikaMoorjani** [reply]
We use either bureau or bureau and banking for underwriting such MSMEs

**Reply by Antriksh-Parmar** [reply]
@KrutikaMoorjani which is anyway available in the other flow right?
